membership/founding-membership.md

```diff
@@ -1,57 +1,42 @@
-# Become a founding member
+# Become a member
 
-We’re currently looking for founding members to help shape the Foundation for Public Code’s long term future by joining us now.
+We’re currently looking for members to help shape the Foundation for Public Code’s long term future.
 
-Founding members are forward-thinking public organizations with a strong principled commitment to the existence of an ecosystem of public code. You’ll be willing to take co-ownership of the Foundation and be an advocate for public code.
+Members are forward-thinking public organizations with a strong commitment to the existence of an ecosystem of public code. As a member you'll help create a global community of public code empowered organizations that pool resources and collaborate in the digital transformation of modern government.
 
-We are looking for organizations that can:
+The Foundation for Public Code was set up to help public organizations develop and maintain software together. This results in higher quality services for the public that are more cost effective, with less risk and more local control.
```

Comment:
Does this need to be included? It feels out of place.

Reply:
I removed 1st sentence and attached the 2nd to the previous paragraph. Future members might react positively to the "less risk" and "more local control" parts.